app/assets/javascripts/Components/test_run_table.jsx

```diff
@@ -176,7 +174,7 @@ class TestScriptResultTable extends React.Component {
   ];
 
   render() {
-    const extraInfo = this.props.data[0]['test_script_results.extra_info'] || '';
```

Comment:
Why did you change the `const` here?

Reply:
I didn't want it to convey some meaning that is really not there.
Are we into using const and let consistently? If we're not, I'd rather just use let for simplicity.

Reply:
I'm trying to use `const` more now (and consistently), so I'd prefer changing this back.